compose/README.md

```diff
@@ -60,9 +60,17 @@ The impact to those following this recipe is that any of the commands below whic
 run as a root user using 'sudo'.
 
 ## Installation
+If you haven't already, create a directory to store this repository using git clone: 
+    
+    $ git clone https://github.com/artefactual-labs/am.git
 
-These are the command you need to run when starting from scratch:
+Run the installation (and all Docker Compose) commands from within the compose directory: 
 
+    $ cd ./am/compose
+
+These are the commands you need to run when starting from scratch:
+
+    $ git pull --rebase
```

Comment:
You don't need `git pull --rebase` when you're starting from scratch because you've just cloned the repo so it's up to date. This is probably not a good idea anyways but it could be considered redundant.